aries-site/src/examples/learn/tutorials/grid-fundamentals-part-1/StepOne.js

```diff
@@ -0,0 +1,43 @@
+// App.js
+import React from 'react';
+import { Grommet, Box, Heading, Text, ThemeContext, Grid } from 'grommet';
+import { hpe } from 'grommet-theme-hpe';
+
+const ProductsPage = () => {
+  return (
+    <>
+      <Heading level={1}>Products</Heading>
+      <Grid columns={{ count: 3, size: 'auto' }}>
+        <Box background="blue" border pad="medium" align="center">
```

Comment:
<img width="997" alt="Screen Shot 2023-04-17 at 10 27 31 AM" src="https://user-images.githubusercontent.com/12522275/232563752-52c457c3-c991-45da-b989-4a39afec2a4c.png">

Can we use a different color? This doesn't pass color contrast

Reply:
Good catch. Thank you. Done.